Library/Contributions/brew_bash_completion.sh

```diff
@@ -600,7 +600,10 @@ _brew ()
     done
 
     if [[ $i -eq $COMP_CWORD ]]; then
-        __brewcomp "$(brew commands --quiet --include-aliases)"
+        # Do not auto-complete "instal" abbreviation for "install" command.
+        # Prefix newline to prevent not checking the first command.
+        local cmds="\n$(brew commands --quiet --include-aliases)"
```

Comment:
The `\n` is not interpreted as you seem to expect. In fact, now the first entry in the list is a literal `\n--cache` instead of `--cache` (just `echo` the string on your command line to check). To get an interpretation like in C (and below) use:

```
$'\n'"$(brew commands --quiet --include-aliases)"
```